django_prices_vatlayer/european_vat.py

```diff
@@ -0,0 +1,34 @@
+from decimal import Decimal
```

Comment:
My point is we're only wrapping a single API and its name is part of the library name. This namespace seems a bit pointless.